packages/engine/src/worker/runner/python/mod.rs

```diff
@@ -256,3 +241,45 @@ async fn _run(
     // }
     Ok(())
 }
+
+struct PythonProcess {
+    _process: tokio::process::Child,
```

Comment:
If we really want a `PythonProcess` (which makes sense), `_process` and shutting down should be handled properly, not simply ignored.